apps/www/src/registry/default/plate-ui/indent-todo-marker-component.tsx

```diff
@@ -18,7 +18,14 @@ export const TodoMarker: MarkerFC = (props) => {
       <Checkbox
         style={{ left: -24, top: 4, position: 'absolute' }}
         checked={element.checked as boolean}
-        onCheckedChange={onChange}
+        onMouseDown={(e: any) => {
+          /**
+           * click the checkbox should not losing the focus.
+           */
+          e.preventDefault();
+          const checked = e.target.getAttribute('aria-checked') !== 'true';
+          onChange(checked);
+        }}
```

Comment:
This should be encapsulated in a prop hook from `@udecode/plate-indent-list`

Reply:
Also, can we keep the `onCheckedChange` prop and just do `preventDefault` inside the `onMouseDown` handler? `onMouseDown` doesn't work with assistive technology.